phpunit.xml

```diff
@@ -1,13 +1,9 @@
 <?xml version="1.0" encoding="UTF-8"?>
 
-<phpunit 
+<phpunit
     bootstrap="vendor/autoload.php"
     colors="true"
-    convertErrorsToExceptions="true"
-    convertNoticesToExceptions="true"
-    convertWarningsToExceptions="true"
-    stopOnFailure="false"
-    syntaxCheck="true"
```

Comment:
As a bonus I cleaned up this file a bit. All of these options were either the [default values](https://phpunit.de/manual/5.7/en/appendixes.configuration.html#appendixes.configuration.phpunit) or [completely nonexistent](https://stackoverflow.com/questions/44328114/phpunit-what-does-syntaxcheck-configuration-parameter-stands-for-exactly).

Reply:
yes, copy paste from old projects...
https://stackoverflow.com/questions/44328114/phpunit-what-does-syntaxcheck-configuration-parameter-stands-for-exactly